package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "orchid",
+  "version": "0.1.0",
```

Comment:
should have `main`, which is entry point for module 😉 
https://docs.npmjs.com/cli/v10/configuring-npm/package-json#main

Reply:
package on this folder orchidui/package.json